Detectors/MUON/MID/Simulation/CMakeLists.txt

```diff
@@ -21,7 +24,10 @@ set(HEADERS
   include/${MODULE_NAME}/ChamberResponseParams.h
   include/${MODULE_NAME}/ColumnDataMC.h
   include/${MODULE_NAME}/Detector.h
+  include/${MODULE_NAME}/Digitizer.h
+  include/${MODULE_NAME}/DigitsMerger.h
```

Comment:
Does the DigitsMerge include need to be exported ?

Reply:
For the moment it is not necessary, but it will soon be when I'll have the DPL workflow for MID reconstruction. The DigitsMerger not only merges the MC digits but also converts them to "data" digits, so that they can be passed to the reconstruction workflow. Since it will be the first device of the workflow, the .h needs to be exported.